app/assets/stylesheets/config/_fonts.scss

```diff
@@ -1,9 +1,9 @@
 // Import Google fonts
-@import url("https://fonts.googleapis.com/css?family=Open+Sans:400,300,700|Raleway:400,100,300,700,500");
+@import url("https://fonts.googleapis.com/css?family=Montserrat:100,300,400,500,600,700,800|Raleway:100,300,400,500,600,700,800);
```

Comment:
I would select less font weights, the more you take the longer it takes to load the page. I would take 2 weights for each font, 3 max if really needed